components/notebook-controller/controllers/notebook_controller.go

```diff
@@ -400,7 +400,21 @@ func generateVirtualService(instance *v1beta1.Notebook) (*unstructured.Unstructu
 	namespace := instance.Namespace
 	clusterDomain := "cluster.local"
 	prefix := fmt.Sprintf("/notebook/%s/%s/", namespace, name)
-	rewrite := fmt.Sprintf("/notebook/%s/%s/", namespace, name)
+	var rewrite string
```

Comment:
I think we should use the _short deceleration operator_ here `:=` instead of `var` since AFAIK `var` also makes the variable have package level scope.

Reply:
Without this, rewrite cannot be referenced in the if statement starting on line 412.

Reply:
@elikatsis Makes sense. Just validated that it still works and pushed the change.